tutorials/interface/activity1.md

```diff
@@ -0,0 +1,244 @@
+
+## Following Tutorials
+
+**⭐Welcome⭐**
+
+You've just figured out the most important part of following a tutorial — reading instructions!
+
+This box is where you'll find information for each step. If you don't find 
+all of the info you need, 
+click on the lightbulb to the right for an extra hint.
+
+When you're ready to move to the next step, click the [ > ] arrow to continue.
+
+
+>>*Tip: If the instructions block too much of the workspace,
+click the* <br />
+**[^ Less...]** <br />
+*button below to close them up.*
+
+#### ~ tutorialhint 
+```
+**You found the hints!**
+```
+
+## Using the workspace
+
+Now let's talk about your [*workspace*](#workIt "The area where you build code").
+
+Your workspace is the area below the instructions where you'll connect blocks to build your program. 
+Not all blocks will connect with one another, but we'll talk more about that later.
+<hr/>
+
+**Try:** Click inside the text area of the ``||game:splash "___"||`` block 
+and change the current sentence to something a little more exciting.
+
+<hr/>
+
+>>*Tip: Did you notice that the first use of the word __workspace__ had a special look? 
+From time to time, we'll enhance important words. Roll your mouse over them to see a definition.*
+
+#### ~ tutorialhint 
+```blocks
+game.splash("I like bananas!")
+```
+
+```template
+game.splash("These blocks are in your workspace!")
+
+```
+
+## Meet the Blocks  @unplugged
+
+Blocks can be dragged out from the toolbox, connected, duplicated, and deleted.
+
+Keep going to learn more about blocks.
+
+![Block Animation](https://raw.githubusercontent.com/microsoft/pxt-arcade/master/docs/static/tutorials/use_blocks.gif "Blocks appear, duplicate, and delete." )
+
+
+## Your Toolbox
+
+The blocks you need won't always be in the workspace when you start. 
+
+In the instructions, block descriptions will often be highlighted 
+in the same color as the category where they live. As an example, we might use the shorthand ``||game:splash "___"||`` when 
+we want you to find this:
+```block
+game.splash(" ")
+```
+
+<hr/>
+
+**Try:** Let's see how this works. Can you find the ``||scene:set background color to [ ]||`` block and add it into the ``||loops:on start||`` block?
+
+#### ~ tutorialhint 
+```blocks
+scene.setBackgroundColor(0)
+game.splash("My monkey is better than yours")
+```
+
+## The Exception
+
+Every rule has an exception, so let's look at a couple of blocks 
+that don't share the same color as the category where they live.
+
+The ``||variables:set my sprite to [ ]||`` and ``||variables:set my projectile to [ ]||`` blocks
+are both red, but they live inside the ``||sprites:Sprites||`` category.
+
+<hr/>
+
+**Try:** Bring a ``||variables:set [my sprite] to [ ] of kind [player]||`` into the ``||loops:on start||`` block and 
+play around with it until your [*sprite*](#sprote "A 2-D image that moves on the screen") shows on the screen.
+
+>>*Tip: Drag the ``||game:splash "___"||`` out of the ``||loops:on start||`` block

```

Comment:
We have a hint macro that you could potentially use
```
### ~ hint

### ~
```

Reply:
I tried that originally, but it didn't do anything in the tutorial-tool, so I didn't know if it was implemented in arcade.